fedimint-server/src/consensus/engine.rs

```diff
@@ -822,8 +822,11 @@ impl ConsensusEngine {
 
             match result {
                 Ok(signed_session_outcome) => return signed_session_outcome,
-                Err(error) => {
-                    tracing::error!(target: LOG_CONSENSUS, "Error while requesting signed session outcome: {}", error);
+                Err(err) => {
+                    // Workaround: timeout messages here are annoying
+                    if !err.to_string().contains("Rpc error: Request timeout") {
+                        warn!(target: LOG_CONSENSUS, %err, "Error while requesting signed session outcome");
+                    }
```

Comment:
This is expected in case we are requesting a session that is still running, right? I'd probably make it a trace level log in case it's a timeout, but also ok with removing.